app/Contracts/DBTable.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace App\Contracts;
+
+/**
+ * Defined table name
+ *
+ * @package App\Contracts
+ */
+class DBTable
+{
+
+    const USER = 'dtb_users';
+    const WEBSITE = 'dtb_websites';
+    const ALERTGROUP = 'dtb_alert_groups';
+    const ALERTMETHOD = 'dtb_alert_methods';
+    const MONITOR = 'dtb_monitors';
+    const ALERTMETHODALERTGROUP = 'dtb_alert_method_alert_groups';
```

Comment:
@khanhpvrk Use readable name for constants. `ALERTMETHODALERTGROUP` => `ALERT_METHOD_ALERT_GROUP`